cmake/modules/MaterialXVersion.rc.in

```diff
@@ -0,0 +1,34 @@
+
+#define MATERIALX_FILEVERSION             @MATERIALX_MAJOR_VERSION@,@MATERIALX_MINOR_VERSION@,@MATERIALX_BUILD_VERSION@,0
+#define MATERIALX_FILEVERSION_STR         "@MATERIALX_MAJOR_VERSION@,@MATERIALX_MINOR_VERSION@,@MATERIALX_BUILD_VERSION@,0\0"
+#define MATERIALX_FILENAME_STR            "@MATERIALX_MODULE_NAME@.dll\0"
+
+1 VERSIONINFO
```

Comment:
I am comparing with the [example version published by Microsoft](https://learn.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource) and a few magic numbers are either replaced by a compiler define or differ in value. Do you think it could be worthwhile to update it to be closer to Microsoft's recommendations?

I have no idea why "040904b0" becomes "040904E4" at line 21 though.

Reply:
OK, the "040904b0"  is a hex representation of lang-codepage, so you declare "U.S. English - Unicode" which is perfectly fine.